commit dd3f052d5c534762c65d47c2978a31e931ad859d
Author: Kai-Heng Feng <kai.heng.feng@canonical.com>
Date:   Thu Feb 16 15:26:54 2017 +0800

    ALSA: hda/realtek - Cannot adjust speaker's volume on a Dell AIO
    
    commit 9f1bc2c4c58fcb2d86e0e26437dc8f3a18ac3276 upstream.
    
    The issue is the same as "dd9aa335c880 ALSA: hda/realtek - Can't adjust
    speaker's volume on a Dell AIO", the output requires to connect to a node
    with Amp-out capability.
    
    Applying the same fixup "ALC298_FIXUP_SPK_VOLUME" can fix the issue.
    
    Signed-off-by: Kai-Heng Feng <kai.heng.feng@canonical.com>
    Signed-off-by: Takashi Iwai <tiwai@suse.de>
    Signed-off-by: Greg Kroah-Hartman <gregkh@linuxfoundation.org>

Index: linuxtest/dummy/rpi_1232_dd3f052d5c534762c65d47c2978a31e931ad859d.txt
===================================================================
--- /dev/null
+++ linuxtest/dummy/rpi_1232_dd3f052d5c534762c65d47c2978a31e931ad859d.txt
@@ -0,0 +1 @@
+dummy file to ensure patch has content.
